src/Faithlife.Analyzers.Vsix/source.extension.vsixmanifest

```diff
@@ -1,22 +1,22 @@
 <?xml version="1.0" encoding="utf-8"?>
 <PackageManifest Version="2.0.0" xmlns="http://schemas.microsoft.com/developer/vsx-schema/2011" xmlns:d="http://schemas.microsoft.com/developer/vsx-schema-design/2011">
-  <Metadata>
-    <Identity Id="Faithlife.Analyzers.ce1e78bc-efbd-41ce-89f5-f1cc932d1ed3" Version="1.0" Language="en-US" Publisher="bgrainger"/>
-    <DisplayName>Faithlife.Analyzers</DisplayName>
-    <Description xml:space="preserve">C# Analyzers used by Faithlife</Description>
-  </Metadata>
-  <Installation>
-    <InstallationTarget Id="Microsoft.VisualStudio.Community" Version="[15.0,)" />
-  </Installation>
-  <Dependencies>
-    <Dependency Id="Microsoft.Framework.NDP" DisplayName="Microsoft .NET Framework" d:Source="Manual" Version="[4.5,)" />
-  </Dependencies>
-  <Assets>
-    <Asset Type="Microsoft.VisualStudio.MefComponent" d:Source="Project" d:ProjectName="Faithlife.Analyzers" Path="|Faithlife.Analyzers|"/>
-    <Asset Type="Microsoft.VisualStudio.Analyzer" d:Source="Project" d:ProjectName="Faithlife.Analyzers" Path="|Faithlife.Analyzers|"/>
-  </Assets>
-  <Prerequisites>
-    <Prerequisite Id="Microsoft.VisualStudio.Component.CoreEditor" Version="[15.0,16.0)" DisplayName="Visual Studio core editor" />
-    <Prerequisite Id="Microsoft.VisualStudio.Component.Roslyn.LanguageServices" Version="[15.0,16.0)" DisplayName="Roslyn Language Services" />
```

Comment:
The upper bound here was preventing me from installing this in my current version of VS (16.10.0). According to this file's history, it looks like this was generated by a template. Let me know if you want me to bump the upper bound to `17.0` instead, or revert the whitespace changes made by the design view GUI.

Reply:
Removing an upper bound seems fine.

It would be nice to have this without whitespace changes.

Reply:
> It would be nice to have this without whitespace changes.

Done: https://github.com/Faithlife/FaithlifeAnalyzers/pull/67/commits/11dbb2fb2003ed80f202e55862128fa1d3d5ef0a#diff-4415222965f0f11e9c08547d1c2edcd46185fe7b511837057e25c312d323da88R3